docker-compose.cpu.yml

```diff
@@ -0,0 +1,406 @@
+# Copyright (c) JSC Intel A/O
```

Comment:
wrong copyright I guess

Reply:
Could you please give me right copyrights?

Reply:
this file has not been updated